os/include/arpa/inet.h

```diff
@@ -189,6 +198,9 @@ uint32_t htonl(uint32_t hl);
 /**
  * @brief Convert the unsigned short integer hostshort from host byte order to network byte order
  *
+ * @details @b #include <arpa/inet.h>\n
+ * SYSTEM CALL API\n
```

Comment:
please check if this API is SYSTEM CALL or not.

Reply:
revised